tests/fixtures/source/toctree/index.rst

```diff
@@ -14,6 +14,11 @@ Toctree
 
     directory/another_file
 
+.. toctree::
+
+    file
+    :ref:`Some specific reference <some_reference>`
```

Comment:
hi @javiereguiluz 
i can't find any examples of such nested ref inside a toctree in the whole symfony docs...

The only thing i can see which looks like this is at the bottom of this file
https://raw.githubusercontent.com/symfony/symfony-docs/5.x/components/event_dispatcher.rst
```rst
.. toctree::
    :maxdepth: 1
    :glob:

    /components/event_dispatcher/*
    /event_dispatcher/*

* :ref:`The kernel.event_listener tag <dic-tags-kernel-event-listener>`
* :ref:`The kernel.event_subscriber tag <dic-tags-kernel-event-subscriber>`
```

(there is another occurrence of the same kind of thing here: https://raw.githubusercontent.com/symfony/symfony-docs/5.x/security/auth_providers.rst)

should we then support this syntax? i'm not even sure sphinx supports it, but I may be wrong...

Reply:
It's OK to remove this use case from the test then 👍  Thanks.